{{ cookiecutter.project_slug }}/Makefile

```diff
@@ -68,6 +70,14 @@ init-data: ## download data
 init-docker: ## initialize docker image
 	$(DOCKER) build -t $(IMAGE_NAME) -f $(DOCKERFILE) --build-arg UID=$(shell id -u) .
 
+sync-to-source: ## sync local data to data source
+{%- if cookiecutter.data_source_type == 's3' %}
+	-aws s3 sync ./data/ $(DATA_SOURCE)
+{%- elif cookiecutter.data_source_type == 'nfs' %}
+	cp -r ./data/* $(DATA_SOURCE)/
```

Comment:
Probably `rsync` is better than `cp`.
Sorry I should have pointed this out when adding nfs source in `init-data`.

Reply:
Thanks for the valuable comment @yoheikikuta 🙏